CMakeLists.txt

```diff
@@ -156,6 +158,9 @@ if ((JPEG_DECODER_FOUND AND NOT WITH_JPEG_DECODER_PLUGIN) OR (JPEG_ENCODER_FOUND
     list(APPEND REQUIRES_PRIVATE "libjpeg")
 endif()
 # TODO: JPEG
+if (OpenJPEG_FOUND AND NOT WITH_OpenJPEG_PLUGIN)
+    list(APPEND REQUIRES_PRIVATE "OpenJPEG")
```

Comment:
Seems to be `libopenjp2` on my system for the pkgconf file.

Reply:
@kmilos: Are you suggesting this instead?
```cmake  
if (OpenJPEG_FOUND AND NOT WITH_OpenJPEG_PLUGIN) 
    list(APPEND REQUIRES_PRIVATE "libopenjp2")

Reply:
Yep, thanks 👍